src/cmp.stub.js

```diff
@@ -1,58 +1,66 @@
-import listener from "./cmp.ssp";
-import Config from "./lib/config";
-import log from './lib/log';
-import {hasLocalVendorConsentCookie} from './lib/cookie/cookie';
-import {GeolocationClient, GDPR_APPLIES_NO} from "./lib/geolocation/client";
-import {GeolocationEventManager} from "./lib/geolocation/eventManager";
+import listener from "./lib/ssp-fallback";
+import config from '../config';
 
-(function(window) {
-	window.ndmCmpConfig = window.ndmCmpConfig || {};
-	const ndmCmpConfig = window.ndmCmpConfig;
-	const configUpdates = {
-		...ndmCmpConfig
-	};
-	Config.update(configUpdates);
-
-	window.ndmCmpGeolocationEventManager = new GeolocationEventManager();
+(function() {
+	const element = document.createElement('script');
+	const firstScript = document.getElementsByTagName('script')[0];
+	const milliseconds = (new Date).getTime();
 
-	if (!hasLocalVendorConsentCookie()) {
-		const geolocationClient = new GeolocationClient(Config.geolocation);
-		geolocationClient.gdprCheck()
-			.then(result => {
-				log.debug(`GDPR check result '${result}'`);
-				window.ndmCmpConfig.gdprApplies = result !== GDPR_APPLIES_NO;
-				window.ndmCmpGeolocationEventManager.gdprCheckDone();
-			});
+	const defaultRequestUrl = 'https://quantcast.mgr.consensu.org/choice.js';
```

Comment:
Since this project uses Webpack and Babel, you could use Axios (https://github.com/axios/axios) or the Fetch API with a polyfill (https://github.com/github/fetch) if you would need to make multiple XHR requests.

Reply:
I will keep those in mind if the code needs to be extended with multiple XHR requests. For now I intend to keep the code as close as possible to the original QuantCast tag.